orocos_kdl/src/CMakeLists.txt

```diff
@@ -103,6 +103,9 @@ ENDIF()
 
 ADD_LIBRARY(orocos-kdl ${LIB_TYPE} ${KDL_SRCS} config.h)
 
+TARGET_INCLUDE_DIRECTORIES(orocos-kdl PUBLIC
```

Comment:
can this line be combined with L134?

Reply:
I just tried to maintain the existing structure as much as possible. And both lines exist for different reasons. So I will keep it as is.